spec/development/workload/PageIterator.ts

```diff
@@ -0,0 +1,61 @@
+import { assert } from "chai";
+import { Event } from "microsoft-graph";
+
+import { PageIterator, PageIteratorCallback } from "../../../src/tasks/PageIterator";
+import { getClient } from "../test-helper";
+
+const client = getClient();
+describe("PageIterator", function() {
+	describe("sameHeadersPassedWithPageIterator", () => {
+		const pstHeader = { Prefer: 'outlook.timezone= "pacific standard time"' };
+		const utc = "UTC";
+		const pst = "Pacific Standard Time";
+
+		it("verify same headers", async () => {
+			const response = await client
+				.api(`/me/events?$top=2`)
+				.headers(pstHeader)
+				.select("id,start,end")
+				.get();
+
+			const callback: PageIteratorCallback = (eventResponse) => {
+				const event = eventResponse as Event;
+				assert.equal(event.start.timeZone, pst);
```

Comment:
is it guaranteed that if the header was not set the timezone would always be UTC hence seeing a different timezone means the header was correctly set?

Reply:
Found this in Graph Docs - [If the Prefer: outlook.timezone header is not specified, the start and end times are returned in UTC](https://github.com/microsoftgraph/microsoft-graph-docs/blob/6cfde54f10f5854c051339437689c15549807824/api-reference/v1.0/api/event-get.md)